Results_LW8.R

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env Rscript
+# Author: OLinares-Perdomo
+# Usage: Results.txt file for LW8 (The ParFastas script and Cecret-wf must be executed before)
+# Parameter: Run Name ;  i.e. Combining_Results.R UT-VH0770-250110
+# last modified on 2025-01-15
+
+library("data.table");library("dplyr");library("utils");
+library("lubridate");find("read.table");library("readr"); 
+
+config<-read_yaml("config.yml")
```

Comment:
Are we going to be editing the config file a lot?

Reply:
I don't anticipate frequent edits to the config file for this particular script. However, if other team members need to add new lines or modify the existing ones, for their scripts, it can be updated as needed to accommodate those changes

Reply:
Is there more utility to have them push all their edits to a config file or to the script itself?